scripts/helpers/assert.js

```diff
@@ -13,6 +13,10 @@ assert.addressEqual = (actual, expected, msg) => {
   assert.equal(toChecksumAddress(actual), toChecksumAddress(expected), msg)
 }
 
+assert.hexEqual = (actual, expected, msg) => {
```

Comment:
The name suggests that the data is hex, should we check for the allowed symbols then?
e.g., smth like `0[xX][0-9a-fA-F]+`